omod/src/main/java/org/openmrs/module/commonlabtest/web/resource/LabTestAttributeTypeResourceController.java

```diff
@@ -54,6 +55,14 @@ public void purge(LabTestAttributeType labTestAttributeType, RequestContext requ
 		throw new ResourceDoesNotSupportOperationException();
 	}
 
+	@Override
+	protected PageableResult doSearch(RequestContext context) {

```

Comment:
Keep in mind that there also exists another method in service to search for test attribute types by name and data type. But since this is out of the scope of this ticket, therefore leaving it for future.